examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<blockquote>the expectation of the cost Hamiltonian is bounded to be not greater than 0.</blockquote><p>Elaborate a bit why it's bounded. For example, is it an intrinsic properties of the Hamiltonian? Or is it something we impose to make training easier?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>